### DISCUSSION TITLE
Hide user@ip:port panel

### DISCUSSION THREAD

**Discussion by ziz0u** [Eugeny/tabby#9869] [General]
How can I hide that panel, where the user and the IP to which I am connected are shown?

![panel-tabby](https://github.com/user-attachments/assets/7ef2f94c-e7db-4110-afa5-6a5be754c062)

Please.

**Comment by Eugeny** [comment]
Yes, there's an "unpin" button on the right side of it

**Reply by ziz0u** [reply]
Thanks very much.
As it took a while to hide, I didn't realize it was working.
